src/main/java/org/sonatype/repository/conan/internal/proxy/matcher/StandardMatcher.groovy

```diff
@@ -0,0 +1,124 @@
+package org.sonatype.repository.conan.internal.proxy.matcher
+
+import org.sonatype.nexus.repository.view.Route.Builder
+import org.sonatype.nexus.repository.view.matchers.ActionMatcher
+import org.sonatype.nexus.repository.view.matchers.token.TokenMatcher
+import org.sonatype.repository.conan.internal.AssetKind
+
+import static org.sonatype.nexus.repository.http.HttpMethods.GET
+import static org.sonatype.nexus.repository.http.HttpMethods.HEAD
+import static org.sonatype.nexus.repository.view.matchers.logic.LogicMatchers.and
+import static org.sonatype.nexus.repository.view.matchers.logic.LogicMatchers.or
+import static org.sonatype.repository.conan.internal.metadata.ConanMetadata.GROUP
+import static org.sonatype.repository.conan.internal.metadata.ConanMetadata.PROJECT
+import static org.sonatype.repository.conan.internal.metadata.ConanMetadata.STATE
+import static org.sonatype.repository.conan.internal.metadata.ConanMetadata.VERSION
+
+/**
+ * For {@link AssetKind#DOWNLOAD_URL} the server sends files in Group/Project/Version order
+ * for all other types it is in Version/Group/Project order
+ * @since 0.0.2
+ */
+class StandardMatcher
+    extends ConanMatcher
+{
+  public static final String NAME = "standard";
+
+  /**
+   * Matches on urls ending with download_urls
+   * @return
+   */
+  Builder downloadUrls() {
+    return new Builder().matcher(
+        and(
+            new ActionMatcher(GET, HEAD),
+            or(
+                downloadUrlsPackagesMatcher(),
+                downloadUrlsMatcher()
+            )
+        )
+    );
+  }
+
+  private static TokenMatcher downloadUrlsMatcher() {
+    return new TokenMatcher("/v1/conans/{${PROJECT}:.+}/{${VERSION}:.+}/{${GROUP}:.+}/{${STATE}:.+}/download_urls");
+  }
+
+  private static TokenMatcher downloadUrlsPackagesMatcher() {
+    return new TokenMatcher("/v1/conans/{${PROJECT}:.+}/{${VERSION}:.+}/{${GROUP}:.+}/{${STATE}:.+}/packages/{sha:.+}/download_urls");
+  }
+
+  /**
+   * Matches on the manifest files
+   * @return
+   */
+  Builder conanManifest() {
+    new Builder().matcher(
+        and(
+            new ActionMatcher(GET, HEAD),
+            or(
+                conanManifestMatcher(),
+                conanManifestPackagesMatcher()
+            )
+        )
+    )
+  }
+
+  private static TokenMatcher conanManifestMatcher() {
+    new TokenMatcher("/{path:.*}/{${PROJECT}:.+}/{${VERSION}:.+}/{${GROUP}:.+}/{${STATE}:.+}/export/conanmanifest.txt")
+  }
+
+  private static TokenMatcher conanManifestPackagesMatcher() {
+    new TokenMatcher("/{path:.*}/{${PROJECT}:.+}/{${VERSION}:.+}/{${GROUP}:.+}/{${STATE}:.+}/package/{sha:.+}/conanmanifest.txt")
+  }
+
+  /**
+   * Matches on conanfile.py
+   * @return
```

Comment:
Another tag with no information, I might go through and check (assuming you keep this code rather than go with the other approach).

Reply:
I would no concern yourself with javadocs just yet. Although it is marked as review, it was really to get some input on my approach to the configurable url.
I will go through a clean up and more tests once the direction is clearer

Reply:
https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/15/commits/fa050ac837279430117d8f757735f1d66da2672e